### DISCUSSION TITLE
Question & Suggeston about Toolbar, Commands and Dockable Panel

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/QRiS#116] [Ideas & Feature Requests] [answered]
For a long time, it hasn't sat well with me that our panels when a project is not open are totally uninviting, unintuitive blank spaces (we inherit this from GCD) and to get them going, you have to find a toolbar (god knows where the user floated it or docked it) with a Riverscapes Consortium logo that you click on.  

![image](https://user-images.githubusercontent.com/9723905/192579819-063d7d1b-9e84-4961-ad6f-9ff9135ee536.png)

I have noticed that most of the more useful panels in Q (not all) have some commands at the top.  Maybe we consider for both QRAVE and QRiS moving the expansive part of the toolbar (i.e. buttons and menu) down to panel? That's where you do most your work anyway. I could be convinced otherwise, or to do both or to leave the "non-Project" stuff up in toolbar. 

![image](https://user-images.githubusercontent.com/9723905/192580417-f04ef42d-9d39-45c6-a74d-da257f62a207.png)

@philipbaileynar and @nick4rivers thoughts?

**Comment by philipbaileynar** [comment, marked as answer]
I am not opposed to your suggestion and its a cheap experiment. 

There is a symantic difference between buttons on the toolbar and dockable windows.

1. Toolbar refers to the map. It contains things you can do to the map in the maps current context (edit session etc). For QRiS these buttons are "global", i.e. above/bigger/more generic than a project.
2. Buttons on a dockable window refer to the context of the dockable window, i.e. in the case of QRiS this means they act on the project and elements within the project.

**Reply by joewheaton** [reply]
So are you thinking @philipbaileynar we can do this experiment in 0.1.6? Do we need an issue for it?
